addons/website/tests/test_automatic_editor.py

```diff
@@ -0,0 +1,27 @@
+from unittest.mock import patch
+
+from odoo.tests import HttpCase, tagged
+
+@tagged('post_install', '-at_install')
+class TestAutomaticEditor(HttpCase):
+
+    def _theme_upgrade_upstream(self):
+        pass
```

Comment:
Why the change from original PR? Not needed anymore to handle actions.todo ?
Just need to pass to prevent some module manipulation? Should be still commented :)